src/Post_Command.php

```diff
@@ -77,7 +77,10 @@ public function __construct() {
 	 *
 	 * [--post_name=<post_name>]
 	 * : The post name. Default is the sanitized post title when creating a new post.
-	 *
+       *
+       * [--from-post=<post_id>]
+       * : The post id of a post to be duplicated.
+       *
```

Comment:
Looks like this is spaces instead of tabs. Can you correct to tabs?

Reply:
yes, I will correct it.